README.md

```diff
@@ -1,20 +1,103 @@
+
+## Introduction
+
+The Mariana Tek Engineering Ladder defines the responsibilities and expectations of people working at various levels within the engineering team. It exists as a rubric for hiring, a guide for engineers seeking excellence and advancement, and a framework for managers to evaluate performance and identify candidates for promotion.
+
+Like many engineering teams, we outline two paths for career advancement: the management track and the technical track. Engineers pursuing the management track (["Managers"](#managers)) are interested in managing people and teams. Engineers pursuing the technical track (["Makers"](#makers)) are interested in honing their technical expertise and shaping the technical direction of the company.
+
+Framed another way by the engineering team at [Chartbeat](https://engineering.chartbeat.com/2015/06/05/engineering-ladders/), the difference between Makers and Managers is how they answer this question:
+
+> “Do I want to build bigger and better systems, or do I want to manage bigger and better teams?”
+
+Every engineer starts their career as a maker. As you work your way from [Engineer I](makers/engineer_i.md) to [Engineer II](makers/engineer_ii.md) to [Senior Engineer](makers/senior_engineer.md), you are primarily focused on expanding your technical knowledge and mastering the skills required of a professional software engineer. When you're ready to move beyond Senior Engineer, you have the opportunity to switch to the management track.
+
+Neither track is superior to the other, and engineers will often switch between them as they move through their careers.
+
+## Makers
+
+* [Engineer I](makers/engineer_i.md)
+* [Engineer II](makers/engineer_ii.md)
 * [Senior Engineer](makers/senior_engineer.md)
-* [Engineer](makers/engineer.md)
+* [Staff Engineer](makers/staff_engineer.md)
+* [Principal Engineer](makers/principal_engineer.md)
+* [CTO](makers/cto.md)
 
-### Managers
+## Managers
 
-* [VP, Engineering](managers/vp_engineering.md)
-* [Director, Engineering](managers/director_engineering.md)
 * [Engineering Manager](managers/engineering_manager.md)
+* [Director, Engineering](managers/director_engineering.md)
+* [VP, Engineering](managers/vp_engineering.md)
 
-### Roles
+## Roles
 
 * [Technical Lead](roles/technical_lead.md)
 * [Directly Responsible Individual (DRI)](roles/directly_responsible_individual.md)
+
+## For New Employees
+
+If you are new to the team, please don't panic while reading the expectations for engineers at your level! Our codebase is huge and complicated, and it's normal to spend 6-12 months on the team before you feel like you know the lay of the land. You will contribute code before then, of course! We do a lot of pair programming and skill-sharing during the onboarding process, and you'll start off with small, well-defined tickets to build up your confidence. Ramping up to your full capabilities will take a while -- and that's okay!
```

Comment:
👍 This is a great piece to read for new employees, it defitintely can be overwhelming looking at our codebase on day one.

Reply:
Being currently in this spot, it is helpful and encouraging to see it in writing. Great that you added that in!